### PR TITLE
Add instructions on GitLab CIViz <> Infra correlation

### PR DESCRIPTION
**Pull request by cg14823**
### What does this PR do?

Adds explanation on how infrastructure and GitLab CI job correlation works.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_pipelines/gitlab.md [hunk 1]
```diff
@@ -139,19 +139,28 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Infrastructure metric correlation
+
+If you are using self-hosted GitLab runners you can correlate jobs to the infrastructure that it is running it.
+For this feature to work the GitLab runner must have a tag of the form `host:<hostname>`. Tags can be added whilst
+[registering a new runner][6] or through the UI, in **Settings > CI/CD > Runners** and editing the appropiate runner,
+or by manually updating the `config.toml` for existing runners.
```
Comment (AdrianLC) [inline content/en/continuous_integration/setup_pipelines/gitlab.md:147]:
```suggestion
[registering a new runner][6], or modifying an already registered one through the Admin UI in **Settings > CI/CD > Runners** or by editing the `config.toml`.
```

File: content/en/continuous_integration/setup_pipelines/gitlab.md [hunk 2]
```diff
@@ -139,19 +139,28 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
...
+
+Once setup CI Visibility will automatically detect the host. To see the metrics just click on a job span in the trace
```
Comment (AdrianLC) [inline content/en/continuous_integration/setup_pipelines/gitlab.md:149]:
```suggestion
After this steps, CI Visibility adds the hostname to each job. To see the metrics click on a job span in the trace
```

File: content/en/continuous_integration/setup_pipelines/gitlab.md (hunk 1 above)
Comment (cg14823) [reply content/en/continuous_integration/setup_pipelines/gitlab.md]: I've reword it a bit so it is clearer

**Review by AdrianLC — APPROVED** [review]

File: content/en/continuous_integration/setup_pipelines/gitlab.md [hunk 3]
```diff
@@ -139,19 +139,28 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Infrastructure metric correlation
+
+If you are using self-hosted GitLab runners you can correlate jobs to the infrastructure that it is running them.
```
Comment (urseberry) [inline content/en/continuous_integration/setup_pipelines/gitlab.md:144]:
```suggestion
If you are using self-hosted GitLab runners, you can correlate jobs with the infrastructure that is running them.
```

File: content/en/continuous_integration/setup_pipelines/gitlab.md [hunk 4]
```diff
@@ -139,19 +139,28 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
...
+For this feature to work the GitLab runner must have a tag of the form `host:<hostname>`. Tags can be added whilst
+[registering a new runner][6]. For existing runners tags can be added by updating the runner's `config.toml` or
+through the UI by going to **Settings > CI/CD > Runners** and editing the appropiate runner.
```
Comment (urseberry) [inline content/en/continuous_integration/setup_pipelines/gitlab.md:147]:
```suggestion
For this feature to work, the GitLab runner must have a tag of the form `host:<hostname>`. Tags can be added while
[registering a new runner][6]. For existing runners, add tags by updating the runner's `config.toml`. Or add tags
through the UI by going to **Settings > CI/CD > Runners** and editing the appropriate runner.
```

File: content/en/continuous_integration/setup_pipelines/gitlab.md [hunk 5]
```diff
@@ -139,19 +139,28 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
...
+
+After this steps, CI Visibility adds the hostname to each job. To see the metrics click on a job span in the trace
+view and in the drawer a new tab named **Infrastructure** will show which contains the host metrics.
```
Comment (urseberry) [inline content/en/continuous_integration/setup_pipelines/gitlab.md:150]:
```suggestion
After these steps, CI Visibility adds the hostname to each job. To see the metrics, click on a job span in the trace
view. In the drawer, a new tab named **Infrastructure** appears which contains the host metrics.
```

**Review by urseberry — APPROVED** [review]

**Pull request merged**